zone/spells.cpp

```diff
@@ -734,6 +735,16 @@ bool Mob::CheckFizzle(uint16 spell_id)
 
 bool Client::CheckFizzle(uint16 spell_id)
 {
+#ifdef LUA_EQEMU
+	bool ignoreDefault = false;
+	bool fizzle = LuaParser::Instance()->CheckFizzle(this, spell_id, spells[spell_id], ignoreDefault);
+
+	if (!fizzle) {
+		return false;
+	} else if (ignoreDefault) {
+		return true;
+	}
```

Comment:
this may be incorrect.
ignoreDefault is a bool to override default behavior with lua or not.
```suggestion
	if (ignoreDefault) {
		return fizzle;
	}
```

Reply:
You are probably right. I'll check it in the morning.

Its a bit of an edge case that I didn't test: when you check fizzle via an override but then still want to use the native function? It's a bit redundant, right?

Reply:
it's how all other mods work. May as well retain the same pattern. Your lua code seems to honor it already, but the C++ integration is pipelined to one use case. Behavior may be abnormal when folks do want a double fizzle check or wahtever odd combination.

Reply:
Did this get followed up on?

Reply:
@alimalkhalifa bump